src/unleash.ts

```diff
@@ -302,18 +306,21 @@ export class Unleash extends EventEmitter {
     if (this.ready) {
       result = this.client.forceGetVariant(name, enhancedContext, fallbackVariant);
     } else {
-      result = typeof fallbackVariant !== 'undefined' ? fallbackVariant : getDefaultVariant();
+      result = typeof fallbackVariant !== 'undefined' ?
+        { ...fallbackVariant, featureEnabled: false } :
+        { ...getDefaultVariant(), featureEnabled: false };
       this.emit(
         UnleashEvents.Warn,
         `Unleash has not been initialized yet. isEnabled(${name}) defaulted to ${result}`,
       );
     }
+    const { featureEnabled, ...variant } = result;
     if (result.name) {
       this.countVariant(name, result.name);
     }
-    this.count(name, result.enabled);
```

Comment:
we were returning variant.enabled not feature enabled

Reply:
Is this the result of `isEnabled` or is this the raw enabled field that comes back from the Unleash API?

Reply:
it wasn't, it was the enabled status of the variant itself.

Reply:
In other SDKs we have isEnabled result in scope so we just use it. Here the isEnabled is called 2 layers deep so we need to propagate it up the stack

Reply:
This PR is doing the following - for enabled and user activated toggle with no variants:
client.getVariant() returns disabled variant and reports enabled toggle metric

Reply:
Thanks! Makes sense!